includes/Story_Post_Type.php

```diff
@@ -304,6 +304,10 @@ public static function admin_enqueue_scripts( $hook ) {
 			$max_upload_size = 0;
 		}
 
+		$preview_query_args = [
+			'preview_id'    => $story_id,
+			'preview_nonce' => wp_create_nonce( 'post_preview_' . $story_id ),
```

Comment:
This seems to come from `post_preview`. Maybe a comment explaining this.

Reply:
Does this nonce need to be refreshed if the browser is open for a while.

Reply:
> Does this nonce need to be refreshed if the browser is open for a while.

Hmm, good point, we should probably update the preview link every time auto-save is called -- it does return a new preview link by default. Perhaps this link shouldn't come from the config at all. _Checking_

Reply:
Added [using updated preview link](https://github.com/google/web-stories-wp/pull/1247/files#diff-4ba4aaa7478f2530815104270cd60e08R118-R120) when opening the preview.